src/history/declarativehistorymodel.cpp

```diff
@@ -54,6 +54,18 @@ void DeclarativeHistoryModel::remove(int index)
     emit countChanged();
 }
 
+void DeclarativeHistoryModel::remove(const QString &url)
+{
+    int index = 0;
+    for (const auto &link: m_links) {
```

Comment:
```suggestion
    for (const auto &link : m_links) {
```
A space missing before colon.